src/layouts/hass-tabs-subpage.ts

```diff
@@ -24,6 +24,7 @@ export interface PageNavigation {
   path: string;
   translationKey?: string;
   component?: string;
+  components?: string[];
```

Comment:
Dashboard entries can have multiple integrations, we show the entry if any are active.